<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Áî®Êà∑ÁôªÂΩï</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background: white;
}

QLabel#titleLabel {
    font-size: 28px;
    font-weight: bold;
    color: #333333;
    margin: 20px 0 10px 0;
}

QLabel#subtitleLabel {
    font-size: 14px;
    color: #666666;
    margin: 0 0 30px 0;
}

QLabel {
    color: #333333;
    font-size: 14px;
    font-weight: normal;
    margin: 0 0 5px 0;
}

QLineEdit {
    padding: 15px;
    border: 1px solid #e0e0e0;
    border-radius: 8px;
    font-size: 14px;
    background: white;
    margin: 0 0 20px 0;
    min-height: 20px;
}

QLineEdit:focus {
    border-color: #667eea;
    outline: none;
}

QPushButton#loginButton {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #667eea, stop:1 #764ba2);
    color: white;
    border: none;
    border-radius: 8px;
    padding: 15px;
    font-size: 16px;
    font-weight: bold;
    margin: 10px 0;
    min-height: 20px;
}

QPushButton#loginButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #5a67d8, stop:1 #6b46c1);
}

QPushButton#loginButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #4c51bf, stop:1 #553c9a);
}

QPushButton#resetButton {
    background: transparent;
    color: #666666;
    border: none;
    padding: 10px 15px;
    font-size: 14px;
    margin: 5px;
}

QPushButton#resetButton:hover {
    color: #333333;
    text-decoration: underline;
}

QPushButton#registerButton {
    background: transparent;
    color: #667eea;
    border: none;
    padding: 10px 15px;
    font-size: 14px;
    margin: 5px;
}

QPushButton#registerButton:hover {
    color: #5a67d8;
    text-decoration: underline;
}

QPushButton#togglePasswordButton {
    background: transparent;
    border: none;
    color: #666666;
    margin: 0;
    padding: 8px;
}

QPushButton#togglePasswordButton:hover {
    background: #f5f5f5;
    border-radius: 4px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>60</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>40</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="headerLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Áî®Êà∑ÁôªÂΩï</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="subtitleLabel">
       <property name="text">
        <string>ËØ∑ËæìÂÖ•ÊÇ®ÁöÑË¥¶Âè∑‰ø°ÊÅØ</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="headerSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="formLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="usernameLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="QLabel" name="usernameLabel">
         <property name="text">
          <string>Áî®Êà∑Âêç:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="usernameEdit">
         <property name="placeholderText">
          <string>ËØ∑ËæìÂÖ•Áî®Êà∑Âêç</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="passwordLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="QLabel" name="passwordLabel">
         <property name="text">
          <string>ÂØÜÁ†Å:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="passwordInputLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="passwordEdit">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>ËØ∑ËæìÂÖ•ÂØÜÁ†Å</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="togglePasswordButton">
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>üëÅ</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="formSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QPushButton" name="loginButton">
       <property name="text">
        <string>ÁôªÂΩï</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="secondaryButtonLayout">
       <item>
        <widget class="QPushButton" name="resetButton">
         <property name="text">
          <string>ÂèñÊ∂à</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="buttonSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="registerButton">
         <property name="text">
          <string>Ê≥®ÂÜå</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="bottomSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>